data/lib/src/network/linshare_http_client.dart

```diff
@@ -200,7 +200,7 @@ class LinShareHttpClient {
   }
 
   AuditLogEntryUserDto? _convertToAuditLogEntryNodeChild(Map<String, dynamic> nodeChildJson) {
-    if (nodeChildJson['type'] == AuditLogEntryType.WORKGROUP.value) {
+    if (nodeChildJson['type'] == AuditLogEntryType.WORK_GROUP.value) {
```

Comment:
please very careful, make sure it still has good interoperability

Reply:
can you add the link?

Reply:
the issue i found was with the version 4 
https://user.linshare-4-2-on-commit.integration-linshare.org

Reply:
What you want to share with this link? `WORKGROUP` & `WORK_GROUP` also work?

Reply:
for newer api v5 and v6 its `WORK_GROUP` and for older api v4 its `WORKGROUP`, other attributes are the same